apps/server/src/modules/instance/repo/instance-repo.service.ts

```diff
@@ -0,0 +1,38 @@
+import { EntityData, EntityName } from '@mikro-orm/core';
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain/types';
+import { BaseDomainObjectRepo } from '@shared/repo/base-domain-object.repo';
```

Comment:
```suggestion
import { BaseDomainObjectRepo } from '@shared/repo';
```

Reply:
Server cant startup with this change. Also leads to dependency cycles because of import of baseEntity -> ALL_ENTITIES